### PR TITLE
E_CLASSROOM-231 Category Add/Edit Form

### PR DESCRIPTION
**Pull request by catampongan**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-231

### Definition of Done
- [x] Add form is correctly displayed according to the Figma design
- [x] Edit form is correctly displayed according to the Figma design
- [x] Card titles changes if user is adding/editing a category
- [x] Button text changes if user is adding/editing a category
- [x] When clicking back button, redirect to list page

### Commands to Run
- N/A

#### Main note
-  Login as an admin user
#### Pages Affected
- http://localhost:3003/admin/add-category
- http://localhost:3003/admin/edit-category

### Scenarios/ Test Cases
- [x] Add form is correctly displayed according to the Figma design
- [x] Edit form is correctly displayed according to the Figma design
- [x] Card titles changes if user is adding/editing a category
- [x] Button text changes if user is adding/editing a category
- [x] When clicking back button, redirect to list page
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
add
edit

### CODE REVIEW

File: src/pages/admin/categories/AddCategory/index.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+          <Form.Label className={style.textDescription}>Description</Form.Label>
+          <Form.Control as="textarea" className={style.inputFieldDescription} />
+        </Form>
+        <Button className={style.button}>Add Category</Button>
+      </Card.Body>
+    </Card>
+  );
 };
 
 export default AddCategory;
```
Comment (karlolee50) [inline src/pages/admin/categories/AddCategory/index.js]: can we make just one AddEditCategory file and pass props to indicate whether we will be adding or editing a category? we can name the file 'CategoryUpsertForm'

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 1

**Comment by Teraaay** [comment]
### Working
- [x] Able to display edit and add category form
- [x] Able to follow the placement of the icons, inputs,  and buttons the same as Figma
- [x] Able to redirect to the Categories list when clicking the back button 

### Comment/s
- [x] The back button on both forms are too big.
- [x] The add and edit form is not displayed in the center [left and right sides of the form have different sizes]. Please display it in the center
- [x] The input fields, buttons, and texts inside the form are too close to the side. Please apply padding.
- [x] The profile icon, name, and navigation links portion of the admin sidebar is not in the position as it was before.

Please refer to the screenshots below.
     
### Screenshot/s
image

image

------
- Total Test Time: 0.25 hr

**Review by karlolee50 — APPROVED** [review]

**Comment by Teraaay** [comment]
### Approved PR #90
- [x] Able to display edit and add category form
- [x] Able to follow the placement of the icons, inputs, and buttons the same as Figma
- [x] Able to redirect to the Categories list when clicking the back button
- [x] Able to fix the form's position, back button size, and form input's spacing.
- [x] Able to fix the admin sidebar,

### Screenshot/s
image

image

----
- Total Test time: 0.5 hr
- Total Iterations: 1
- Total Comments: 2

**Pull request merged**